### PR TITLE
fix: Empty state text

### PR DESCRIPTION
**Pull request by joethreepwood**
## Problem

screenshot_2023-07-19_at_12 56 25

As @annikaschmid pointed out, the text doesn't work for all contexts. 

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/lib/components/ProductIntroduction/ProductIntroduction.tsx [hunk 1]
```diff
@@ -77,8 +77,7 @@ export const ProductIntroduction = ({
                     <p className="ml-0">{description}</p>
                     {!isEmpty && (
                         <p className="ml-0">
-                            Your team has already started using {productName}. You can jump in to see what your team has
-                            made, or create a new one yourself.
+                            Your team is already using {productName}. You can take a look at what they're  doing, or get started yourself.
```
Comment (joethreepwood) [inline frontend/src/lib/components/ProductIntroduction/ProductIntroduction.tsx:80]:
```suggestion
                            Your team is already using {productName}. You can take a look at what they're doing, or get started yourself.
```

**Review by benjackwhite — APPROVED** [review]
Nice @joethreepwood !!

**Comment by joethreepwood** [comment]
I think we can still merge this in, but totally unclear on why the check is failing.

**Comment by joethreepwood** [comment]
@raquelmsmith I'll let this close for now as I couldn't get the checks to pass, but FYI on the suggested fix for the language

**Comment by raquelmsmith** [comment]
@joethreepwood the issue here was just formatting. Install the Prettier extension in VS code and it will solve all (and I mean _all_) of your problems. 😉

**Pull request merged**